manifests/0000_31_cluster-baremetal-operator_05_rbac.yaml

```diff
@@ -71,13 +71,24 @@ metadata:
   creationTimestamp: null
   name: cluster-baremetal-operator
 rules:
-- resources:
-  - events
+- apiGroups:
```

Comment:
I expect all of these changes need to be made in files under `config/rbac` instead of directly to this generated file.

Reply:
I am not sure if that is the complete picture because "deployments" are not specified in any yaml files in config/rbac but is present in manifests/0000_31_cluster-baremetal-operator_05_rbac.yaml.

Reply:
Isn't that in config/rbac/role.yaml?

Reply:
I hadn't realized https://github.com/openshift/cluster-baremetal-operator/pull/52 had merged overnight. That explains all that I was seeing.